frontend/src/components/HistoryPannel/ChatHistoryPanel.tsx

```diff
@@ -0,0 +1,25 @@
+import { AddFilled } from "@fluentui/react-icons"
+
+import styles from "./ChatHistoryPannel.module.css";
+
+export const ChatHistoryPanel = () => {
+    return (
+        <section className="container" data-is-scrollable aria-label="chat history panel" style={{ marginRight: 10, marginTop: 5, width: 250, padding: "0 10px"}}>
```

Comment:
@KenyerRamirez shouldn't  these inline styles be included into the ChatHistoryPannel.module.css